test/routes/v1.test.ts

```diff
@@ -904,6 +904,26 @@ const UTILITIES = [
       },
     },
   ],
+  [
+    '02116',
+    {
+      location: { state: 'MA', city: 'Boston', county_fips: '25025' },
+      utilities: {
+        'ma-eversource': {
+          name: 'Eversource',
+        },
+      },
+      gas_utilities: {
+        'ma-national-grid-gas': {
+          name: 'National Grid',
+        },
+        'ma-nstar-gas-company': {
+          name: 'Eversource',
+        },
+      },
+      gas_utility_affects_incentives: true,
```

Comment:
So the gas utilities will always be determined and returned, and this flag will just be used to let the client decide whether to show them or not?

Reply:
Right